### ISSUE TITLE
Allow use of alias for foreign composers (ie, Russian)

### ISSUE THREAD

**Issue by jeff47** [Lidarr/Lidarr#3481]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Is your feature request related to a problem? Please describe

I have a number of Russian composers in my library, for example Сергей Рахманинов.  (Sergei Rachmaninoff)  Not being Russian and not having a cyrillic keyboard, this is not trivial to try and find in my library.  Nor is it the easiest thing to distinguish between different Russian artists.

### Describe the solution you'd like

Allow use of romanized names (either from aliases, or potential "sort name"?)

### Describe alternatives you've considered

Learn Russian?  

### Anything else?

nope

**Comment by SamLawAnthro** [comment]
This has been driving me nuts! Such an easy fix, Musicbrainz has english language alias in their artist data so it could just be a small tweak to the API

**Comment by madutis** [comment]
I've encountered this issue while trying to organize albums from Youn Sun Nah, Ryuichi Sakamoto, Joe Hisaishi where Lidarr creates folders in Kanji that I can't read or search for. So strange this hasn't been an issue for many more people.

Musicbrainz has a Sort Name variable that has a romaji name for all these artists. Please can we have that as a variable in media management?

**Comment by GabrielPearce** [comment]
This is something that I have just ran into as well, is there some sort of limitation that would make this not possible? this would be an amazing feature

**Comment by thomaslazar** [comment]
I would really like to have this issue solved as well.

**Comment by natsuklsubaru** [comment]
I am willing to work up a PR for this. MusicBrainz relies on Pseudo Releases for alternate titles and aliases for artist names. Devs/maintainers, please let me know what would be a good way to implement this.

Thanks.

**Comment by aglowinthefield** [comment]
> This is something that I have just ran into as well, is there some sort of limitation that would make this not possible? this would be an amazing feature

Taking a look at this as well, and it seems like it might need a metadata API change as well. The DB schema (as far as I can tell) only stores aliases as lists of strings, so there's not really a way that I can see to map it back to a given language.

**Comment by Kamikaze01git** [comment]
Hello Lidarr team and fans :)

I'm going crazy here as well, as I also have some artists whose names are displayed only in Asian or Cyrillic characters.
It's driving me absolutely insane.
Even "Sergei Rachmaninoff" is called "Сергей Васильевич Рахманинов" in Lidarr (next to many others...).

This is not only a problem in the filesystem because of all the special (asian,...) characters, but also with the ID tags and when searching for the artists.
Unfortunately, I wasn’t able to find any way to edit this manually in Lidarr. Maybe I overlooked something, but at the moment I have no idea where I could manually set the artist name, for example.

Lidarr is really good and powerful — but this naming issue really hurts a lot :(

**Comment by aglowinthefield** [comment]
> Hello Lidarr team and fans :)
> 
> I'm going crazy here as well, as I also have some artists whose names are displayed only in Asian or Cyrillic characters. It's driving me absolutely insane. Even "Sergei Rachmaninoff" is called "Сергей Васильевич Рахманинов" in Lidarr (next to many others...).
> 
> This is not only a problem in the filesystem because of all the special (asian,...) characters, but also with the ID tags and when searching for the artists. Unfortunately, I wasn’t able to find any way to edit this manually in Lidarr. Maybe I overlooked something, but at the moment I have no idea where I could manually set the artist name, for example.
> 
> Lidarr is really good and powerful — but this naming issue really hurts a lot :(

Yeah, you currently can't do it because the metadata API doesn't support it. It will require a metadata DB schema update as well since aliases are currently only stored as a list of Strings, so there'd be no way to target your specific locale even if they were exposed :/ I keep meaning to try and get to this but at least I can offer the 'why' haha

**Comment by dcicco** [comment]
I opened #5604 to hopefully cover this!

**Comment by grodius** [comment]
are the devs just ghosting? it’s wild it’s not already in, when you can add an artist based on the alias, and this PR is collecting dust
